librz/util/str.c

```diff
@@ -3819,3 +3819,71 @@ RZ_API char *rz_str_version(const char *program) {
 }
 
 #undef RZ_STR_PKG_VERSION_STRING
+
+/**
+ * \brief Converts a raw buffer to a printable string based on the selected options
+ *
+ * \param  buf          The buffer to read
+ * \param  len          The length of the buffer
+ * \param  max_wrap_len The max line length when RZ_STR_STRINGIFY_WRAP is set
+ * \param  options      The options flags to follow to generate the output.
+ * \return The stringified raw buffer
+ */
+RZ_API RZ_OWN char *rz_str_stringify_raw_buffer(RZ_NONNULL const ut8 *buf, ut32 len, ut32 options, ut32 max_wrap_len) {
+	rz_return_val_if_fail(buf && len > 0, NULL);
```

Comment:
```suggestion
	rz_return_val_if_fail(buf && len >= 0, NULL);
```
I think having len=0 is a valid case when the API should not do anything.